cscs-checks/apps/greasy/greasy_check.py

```diff
@@ -234,3 +233,24 @@ def num_ranks(match):
         ))
 
         return True
+
+    @run_before('performance')
+    def set_perf_patterns(self):
```

Comment:
What about using the new `@performance_function` decorator that was just merged?